app/models/edition.rb

```diff
@@ -30,6 +30,9 @@ class Edition < ActiveRecord::Base
   has_many :classification_featurings, inverse_of: :edition
   has_many :links_reports, as: :link_reportable
 
+  has_many :dependencies, class_name: 'EditionDependency', foreign_key: :edition_id, dependent: :destroy
```

Comment:
Hi David: as i mentioned here: https://github.com/alphagov/whitehall/pull/2009#commitcomment-10045409, `has_many :edition_dependencies` needs to be reserved for being used `as: :dependable`, the same way as we use in [`Contact`](https://github.com/alphagov/whitehall/pull/2009/files#diff-c4b0c8c9c2b6974a9038fdb3f22c11c4R6).